src/core/filters/packets/bpf/stub.bpf.c

```diff
@@ -0,0 +1,8 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#include <packet-filter.h>
+
+__attribute__ ((noinline))
+int packet_filter(struct filter_context *ctx)
+{
+	return (volatile int)0;
```

Comment:
Uhm, no, it should work :) I guess. Initially, it was a variable, but for some reason remained like this.
But "volatile" here is not needed as we just need to emit the code for the function and the BTF sections. This will never be called